core/META-INF/MANIFEST.MF

```diff
@@ -139,7 +139,7 @@ Bundle-ClassPath: .,
  lib/commons-lang3-3.8.1.jar,
  lib/xstream-1.4.10.jar,
  lib/xpp3-1.1.4c.jar,
- lib/picocontainer-2.11.2.jar,
+ lib/picocontainer-2.11.2-patched_relocated.jar,
```

Comment:
So is this jar already patched or just repacked because the former approach was using classpath reordering so that the original class file was not loaded from the library but the patched one was loaded from our core library.

Reply:
This jar contains the patched classes.